installer.sh

```diff
@@ -211,7 +212,7 @@ print_help()
   echo -e "\\n""$CYAN""USAGE""$NC"
   echo -e "$CYAN""-d""$NC""         Default installation of all dependencies needed for EMBA in default/docker mode (typical initial installation)"
   echo -e "$CYAN""-F""$NC""         Installation of EMBA with all dependencies (for running on your host - developer mode)"
-  echo -e "$CYAN""-c""$NC""         Complements EMBA dependencies (get/install all missing files/applications)"
+#  echo -e "$CYAN""-c""$NC""         Complements EMBA dependencies (get/install all missing files/applications)"
```

Comment:
Why are you changed this line to a comment? `-c` is a valid option at the moment and if don't want to support it anymore, then we should remove it completely from the installer.

Reply:
As discussed: We will handle with it in the next update - `-c` will be completely removed, because it isn't needed anymore.